Unable to install packages after Migration to React v18

Are there going to be any breaking changes after updating React to v18 in peer dependencies?
Currently, I'm unable to install `tss-react` in my React v18 project:

```
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0 || ^17.0.2" from tss-react@3.6.0
npm ERR! node_modules/tss-react
npm ERR!   tss-react@"^3.6.0" from the root project
```

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     ],
     "homepage": "https://www.tss-react.dev",
     "peerDependencies": {
-        "react": "^16.8.0 || ^17.0.2",
+        "react": "^16.8.0 || ^17.0.2 || ^18.0.0",
         "@emotion/react": "^11.4.1"
     },
     "dependencies": {

